cmd/tailscaled/tailscaled.go

```diff
@@ -55,6 +55,9 @@ import (
 	"tailscale.com/wgengine/monitor"
 	"tailscale.com/wgengine/netstack"
 	"tailscale.com/wgengine/router"
+
+	// Initialize the aws store.
+	_ "tailscale.com/ipn/store/aws"
```

Comment:
why is kube and aws not on equal footing? they should probably both require importing.

but if you use tsnet and forget one, the built-in package should know about their prefixes enough to tell you a useful error like, "Yo, you forgot to import _ "..../kubee""

Reply:
I moved KubeStore to it's own package, and added add a generic `tailscale.com/ipn/store` package to instantiate all the stores so that it's all in one place.